docs/peparing-your-session.md

```diff
@@ -0,0 +1,82 @@
+# Before the stream
+
+Delivering an online session involves a little more than taking one you've done in person and talking into a camera. You want to consider the environment from which you'll be speaking, make sure it fits the general tone of the Reactors, and you account for questions.
+
+## Physical environment
+
+While you can stream from any room you like, there are a couple of core minimum requirements you will want to consider.
+
+- Ensure you are lit from above or in front of you; back-lighting will cause you to be too dark
+  - Special lighting is not required, but you may want to consider it (details below)
+- You will need a quiet room, preferably with a door
+  - Soundproofing not needed
+- Have a clean backdrop behind you; again, nothing fancy
+  - Feel free to show some personality!
+    - Bookshelves with books and other nick-knacks are great!
+  - If you want to invest in a green screen you can, but this is not needed
+
+## Online presentations
+
+Delivering an online presentation focuses on different skills than an in person event. You will want to take a little time to update your presentation and skill set before going live.
+
+### Talking into the void
+
+Unlike an in-person event, you won't receive immediate feedback from your audience about how things are going. This can be a bit unsettling the first time you deliver a session, but it will become more comfortable with experience.
+
+- You may find your timing will be different than an in-person event
+  - Some speakers talk faster because there is no audience to interrupt them
+  - Some speakers talk more because there is no audience to interrupt them
+- Practice presenting to the camera
+  - Will help get you comfortable talking to a camera
+    - Remember you will be providing all of the energy for the presentation
+  - Will help you get accustomed to your pacing
+
```

Comment:
Might be good to add something about the delay. Ex:
- Rememer that there is a slight delay from when you speak and when it is broadcasted to the audience. Typically 20-30 seconds

Reply:
Oh, but you do add that a bit down below :)